ocean_provider/routes/decrypt.py

```diff
@@ -42,6 +41,62 @@
 @services.route("/decrypt", methods=["POST"])
 @validate(DecryptRequest)
 def decrypt():
+    """Decrypts an encrypted document based on transaction Id or dataNftAddress.
+
+    ---
+    consumes:
+      - application/json
+    parameters:
+      - name: decrypterAddress
+        description: address of agent requesting decrypt
+        type: string
+        required: true
+      - name: chainId
+        description: chainId selection for decrption
+        type: int
+        required: true
+      - name: transactionId
+        description: transaction Id for document,
+            required if dataNftAddress, encryptedDocument and flags parameters missing
```

Comment:
I think this could be improved to `transaction id where the document was created or last updated`